<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Win32 Ln</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCommandLinkButton" name="buttonFileSymbol">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>创建文件符号链接</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="buttonFileHard">
      <property name="text">
       <string>创建文件硬链接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="buttonDirSymbol">
      <property name="text">
       <string>创建目录符号链接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="buttonDirJunction">
      <property name="text">
       <string>创建目录联接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
